### DISCUSSION TITLE
How to  set  Min as default browser?

### DISCUSSION THREAD

**Discussion by AhzDeveloperlol** [minbrowser/min#2181] [Q&A]
How do I set Min as the default browser?

**Comment by unrealbanter** [comment]
The only way I've found is to set it on your computer's preferences by default.

**Comment by mikeyww** [comment]
I use the following registry settings to register the browser in Windows as **%LOCALAPPDATA%\min\min.exe**.

I think "DefaulIcon" should be "DefaultIcon".

`Windows Registry Editor Version 5.00

[HKEY_CLASSES_ROOT\Min]
@="Min Browser Document"
[HKEY_CLASSES_ROOT\Min\Application]
[HKEY_CLASSES_ROOT\Min\DefaulIcon]
[HKEY_CLASSES_ROOT\Min\shell]
[HKEY_CLASSES_ROOT\Min\shell\open]
[HKEY_CLASSES_ROOT\Min\shell\open\command]
@=hex(2):22,00,25,00,4c,00,4f,00,43,00,41,00,4c,00,41,00,50,00,50,00,44,00,41,\
  00,54,00,41,00,25,00,5c,00,6d,00,69,00,6e,00,5c,00,6d,00,69,00,6e,00,2e,00,\
  65,00,78,00,65,00,22,00,20,00,22,00,25,00,31,00,22,00,00,00

[HKEY_CURRENT_USER\Software\Classes\Applications\min.exe]
[HKEY_CURRENT_USER\Software\Classes\Applications\min.exe\shell]
[HKEY_CURRENT_USER\Software\Classes\Applications\min.exe\shell\open]
[HKEY_CURRENT_USER\Software\Classes\Applications\min.exe\shell\open\command]
@=hex(2):22,00,25,00,4c,00,4f,00,43,00,41,00,4c,00,41,00,50,00,50,00,44,00,41,\
  00,54,00,41,00,25,00,5c,00,6d,00,69,00,6e,00,5c,00,6d,00,69,00,6e,00,2e,00,\
  65,00,78,00,65,00,22,00,20,00,22,00,25,00,31,00,22,00,00,00

[HKEY_LOCAL_MACHINE\SOFTWARE\Classes\Min]
[HKEY_LOCAL_MACHINE\SOFTWARE\Classes\Min\shell]
[HKEY_LOCAL_MACHINE\SOFTWARE\Classes\Min\shell\open]
[HKEY_LOCAL_MACHINE\SOFTWARE\Classes\Min\shell\open\command]
@=hex(2):22,00,25,00,4c,00,4f,00,43,00,41,00,4c,00,41,00,50,00,50,00,44,00,41,\
  00,54,00,41,00,25,00,5c,00,6d,00,69,00,6e,00,5c,00,6d,00,69,00,6e,00,2e,00,\
  65,00,78,00,65,00,22,00,20,00,22,00,25,00,31,00,22,00,00,00`
